src/transformers/generation/logits_process.py

```diff
@@ -193,12 +193,49 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
 
 class RepetitionPenaltyLogitsProcessor(LogitsProcessor):
     r"""
-    [`LogitsProcessor`] enforcing an exponential penalty on repeated sequences.
+    [`LogitsProcessor`] prevents the repetition of previous tokens through an exponential penalty. 
+    This technique invloves adjusting the scores assigned to previously generated tokens, discouraging repetition. 
```

Comment:
```suggestion
```
(all logits processors adjust the scores, so this sentence doesn't add information :) )